x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/index.tsx

```diff
@@ -126,12 +129,23 @@ export class Waterfall extends Component<Props> {
   };
 
   public render() {
-    const { waterfall } = this.props;
+    const { waterfall, exceedsMax } = this.props;
     const itemContainerHeight = 58; // TODO: This is a nasty way to calculate the height of the svg element. A better approach should be found
     const waterfallHeight = itemContainerHeight * waterfall.orderedItems.length;
 
     return (
       <Container>
+        {exceedsMax ? (
+          <EuiCallOut
+            color="warning"
+            size="s"
+            iconType="alert"
+            title={i18n.translate('xpack.apm.waterfall.exceedsMax', {
+              defaultMessage:
+                'Number of items in this trace exceed what is displayed'
```

Comment:
I think this is a little cryptic if you don't know what we are talking about. Ideally we'd let the user know how many trace items are being left out like: 
"Only the first 1000 items in a trace will be displayed, and 293 items were therefore left out ". Although having those two variables (1000 and 293) will be a bit of a hassle so I'd rather not do that. 

Perhaps @bmorelli25 has an idea on how to improve this.

Reply:
Hmmm, I'm struggling on this one. Without using variables I'm not sure we can do much better here.

Reply:
Variables aren't completely off the table for messaging here, but it does require some extra data being passed to support it. If the only useful messaging requires these variables, I can make it work.

Reply:
I'm good with it if @bmorelli25 is :)

Reply:
I'm good with it as well. We can always improve it later if need be.